crates/api/src/instance.rs

```diff
@@ -182,8 +170,15 @@ impl Instance {
     /// export you'll need to consult [`Module::exports`]. The list returned
     /// here maps 1:1 with the list that [`Module::exports`] returns, and
     /// [`ExportType`](crate::ExportType) contains the name of each export.
-    pub fn exports(&self) -> &[Extern] {
-        &self.exports
+    pub fn exports<'me>(&'me self) -> impl Iterator<Item = Extern> + 'me {
```

Comment:
To enhance this a bit, perhaps this could return an `ExactSizeIterator` which should allow for enumeration and such?

Reply:
Works great.